src/app/components/apparatus-entry/apparatus-entry-readings/apparatus-entry-readings.component.html

```diff
@@ -20,6 +20,6 @@
     <span class="font-italic" *ngIf="el.content.length === 0">omit.</span>
     <!-- TODO: handle lacunastart and lacunaend -->
     <ng-container *ngFor="let witID of el.witIDs">
-        <span *ngFor="let wit of getWits$(witID) | async" class="font-italic"> {{ wit }} </span>
+        <span *ngFor="let wit of getWits$(witID) | async" class="font-italic appa-witn"> {{ wit }} </span>
```

Comment:
Can you find a better name for this class? Maybe "app-wits"? We use "app" and "wits" in the project